fetchai/ledger/serialisation/transaction.py

```diff
@@ -141,7 +142,25 @@ def encode_payload(buffer: io.BytesIO, payload: transaction.Transaction):
         identity.encode(buffer, signer)
 
 
-def encode_transaction(payload: transaction.Transaction, signers: List[crypto.Entity]):
+def encode_multisig_transaction(payload: 'Transaction', signatures: Dict[Identity, bytes]):
```

Comment:
encode_multisig_transaction is not stepped through by any unit test

Reply:
I put debug statement in and ran the unit tests, and it did not stop. Crucial method so must be tested.